src/components/donation-flow/DonationFlowForm.tsx

```diff
@@ -58,17 +115,32 @@ export function DonationFlowForm() {
                 marginRight: 'auto',
               }}
               autoComplete="off">
-              <StepSplitter content="1" />
-              <Amount />
-              <StepSplitter content="2" />
-              <PaymentMethod />
-
-              <SubmitButton>Submit</SubmitButton>
+              <Box mb={2}>
+                <StepSplitter content="1" active={Boolean(values.amount)} />
+                <Amount />
+                <StepSplitter
+                  content="2"
+                  active={Boolean(values.amount) && Boolean(values.payment)}
```

Comment:
@dimitur2204 I think `Boolean(values.payment)` will always resolve to `true` as it checks `Boolean('card')` or `Boolean('bank')`